indy_client/test/cli/tmp_wallet_restore_issue

```diff
@@ -0,0 +1 @@
py/tuple": ["degree", "Bachelor of Science, Marketing"]}, {"py/tuple": ["status", "graduated"]}, {"py/tuple": ["ssn", "123-45-6789"]}]}]}, "ts": null, "version": "0.2", "verifiableAttributes": {"777fb1af-a7a0-4f0c-81b8-9e09a8068d16": {"py/object": "anoncreds.protocol.types.AttributeInfo", "py/newargs": {"py/tuple": ["status", null, null]}, "py/seq": ["status", null, null]}, "d5743fcc-d287-4176-a634-3b721d962ea5": {"py/object": "anoncreds.protocol.types.AttributeInfo", "py/newargs": {"py/tuple": ["ssn", null, null]}, "py/seq": ["ssn", null, null]}, "4de5205f-720f-49f0-8397-0576875d904b": {"py/object": "anoncreds.protocol.types.AttributeInfo", "py/newargs": {"py/tuple": ["degree", null, null]}, "py/seq": ["degree", null, null]}}, "nonce": 1871218719015472932666560146158750511756, "seqNo": null, "name": "Job-Application", "predicates": {}, "fulfilledByClaims": [{"py/tuple": [{"py/id": 12}, {"py/id": 14}, {"py/reduce": [{"py/type": "collections.OrderedDict"}, {"py/tuple": []}, null, null, {"py/tuple": [{"py/tuple": ["student_name", "Alice Garcia"]}, {"py/tuple": ["ssn", "123-45-6789"]}, {"py/tuple": ["degree", "Bachelor of Science, Marketing"]}, {"py/tuple": ["year", "2015"]}, {"py/tuple": ["status", "graduated"]}]}]}]}], "selfAttestedAttrs": {}}], "internalId": null, "remoteEndPoint": {"py/tuple": ["10.0.0.203", 6666]}, "request_nonce": "57fbf9dc8c8e6acde33de98c6d747b28c", "name": "Acme Corp", "connection_status": null, "trustAnchor": "Acme Corp", "verifiedClaimProofs": [], "py/object": "indy_client.client.wallet.connection.Connection", "connection_last_sync_no": null, "localIdentifier": "Siga5PyLFTZdpupPUXogjt", "localVerkey": "~9wzQSoSNbLwRcNz9JkBFME", "remotePubkey": "C5eqjU7NMVMGGfGfx2ubvX5H9X346bQt5qeziVAo3naQ", "availableClaims": [], "connection_last_synced": {"py/object": "datetime.datetime", "__reduce__": [{"py/type": "datetime.datetime"}, ["B+EIChQsHwmWIA=="]]}, "_remoteVerkey": "~WjXEvZ9xj4Tz9sLtzf7HVP", "remoteIdentifier": "CzkavE58zgX7rUMrzSinLr"}]}]}]}, "replyHandler": {}, "knownIds": {"ULtgFQJe6bjiFbs7ke3NJD": {"py/object": "indy_common.identity.Identity", "identity": {"py/object": "plenum.common.signer_did.DidIdentity", "abbreviated": null, "_identifier": "ULtgFQJe6bjiFbs7ke3NJD", "_verkey": null}, "last_synced": null, "seqNo": null, "trustAnchor": null, "_role": null}, "JYeHd6Zn3zFo1UbNVBd6U1": {"py/object": "indy_common.identity.Identity", "identity": {"py/object": "plenum.common.signer_did.DidIdentity", "abbreviated": null, "_identifier": "JYeHd6Zn3zFo1UbNVBd6U1", "_verkey": null}, "last_synced": null, "seqNo": null, "trustAnchor": null, "_role": null}, "CzkavE58zgX7rUMrzSinLr": {"py/object": "indy_common.identity.Identity", "identity": {"py/object": "plenum.common.signer_did.DidIdentity", "abbreviated": null, "_identifier": "CzkavE58zgX7rUMrzSinLr", "_verkey": null}, "last_synced": null, "seqNo": null, "trustAnchor": null, "_role": null}}, "lastKnownSeqs": {}, "env": "no-env", "classver/indy_client.client.wallet.wallet.Wallet": 1}
```

Comment:
We should keep the old version of wallet as well to make sure that wallets are backward-compatible

Reply:
There are other tests for verification of deserialization of wallets in formats of previous versions.